alda-mode.el

```diff
@@ -277,6 +277,29 @@ Because alda runs in the background, the only way to do this is with alda restar
   ;; Set alda highlighting
   (setq font-lock-defaults '(alda-highlights)))
 
+(defun alda-play-block ()
+  (interactive)
+  (let* ((p (point)))
```

Comment:
I think `save-excursion` could save setting the point and doing a goto later and be cleaner